.circleci/config.yml

```diff
@@ -321,18 +317,8 @@ workflows:
             branches:
               only: main
 
-  manual-trigger-release:
+  manual-trigger-bump:
     when:
-      not:
-        equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
+      equal: [ bump, << pipeline.parameters.action >> ]
     jobs:
-      - trigger-automatic-release:
-          type: approval
-          filters:
-            tags:
-              ignore: /.*/
-            branches:
-              only: main
-      - automatic-release:
-          requires:
-            - trigger-automatic-release
+      - automatic-release
```

Comment:
It's a bit strange that a "manual" workflow to trigger an "automatic" job... Wonder if this should just be `release`... Same with the dependency update job. NABD though.